### PR TITLE
Src

### PR DESCRIPTION
**Pull request by vegardlekve**
[x] Create a design or wireframe for each page
[x] Create a new public repository on GitHub
[x] Create a new project folder for this repository
[x] Create a new git branch for this project
[x] Create a package.json file
[x] Install Bootstrap, SASS and Live Server
[x] Configure NPM scripts in package.json
[x] Implement your design using Bootstrap and SASS overrides

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Happy birthday, you son of a gun!</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend5.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```
Comment (sulenchy) [inline index.html]: You would want to add a valid value for alt attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                <div class="card-body">
+                    <p class="card-text">Congratulations!</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend3.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```
Comment (sulenchy) [inline index.html]: add value for alt attribute here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                <div class="card-body">
+                    <p class="card-text">Getting old! Although the alternative is worse.. HBD :D</p>
+                    <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```
Comment (sulenchy) [inline index.html]: same thing here and in other places

File: login.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign in</title>
+    <link rel="stylesheet" href="/dist/css/styles.css">
+</head>
+
+<body class="text-center d-flex flex-column">
+
```
Comment (sulenchy) [inline login.html:42]: Shouldn't you have the navbar here

File: profile.html [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Profile page</title>
+    <link rel="stylesheet" href="/dist/css/styles.css">
+</head>
+
+<body>
+    <main>
```
Comment (sulenchy) [inline profile.html]: Any reason why the navbar is inside the main element? I think the navbar is good enough to be on its own. It is semantically incorrect.

File: profile.html [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Happy birthday, you son of a gun!</p>
+                        <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend5.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```
Comment (sulenchy) [inline profile.html]: Why do you have images in the dist folder? Static files such as images, favicons, fonts, etc are meant to be kept in public. So I suggest you do:

- create a public folder in the root folder
- move the images folder in the dist folder into the public folder
- fix the path in diff places in your html files

File: src/scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,59 @@
+@import "variables";
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+/* Bootstrap Sign In Example */
+
+.form-signin {
```
Comment (sulenchy) [inline src/scss/styles.scss:7]: Good job! but then all these styles are just CSS in an SCSS file. Your style would be neater with pure scss like

```
.form-signin {
    max-width: 400px;
    padding: 15px;

    .form-floating:focus-within {
          z-index: 2;
     }
```

Read more at https://noroff-content.gitlab.io/feu/css-frameworks/intro-to-sass.html#nesting

**Review by sulenchy — COMMENTED** [review]
@vegardlekve left some important comments for you above. Please work on it

File: index.html (hunk 1 above)
Comment (vegardlekve) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (vegardlekve) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (vegardlekve) [reply index.html]: fixed

File: login.html (hunk 4 above)
Comment (vegardlekve) [reply login.html:42]: Yes, that didn't make sense. Fixed.

File: profile.html (hunk 5 above)
Comment (vegardlekve) [reply profile.html]: No other reason then summer rustiness. Fixed.

File: profile.html (hunk 6 above)
Comment (vegardlekve) [reply profile.html]: Not sure what I was thinking placing them in dist. Fixed as suggested.

File: src/scss/styles.scss (hunk 7 above)
Comment (vegardlekve) [reply src/scss/styles.scss:7]: Fixed as suggested. Removed classed that wasn't used. Not sure how nest inputs.

File: index.html (hunk 3 above)
Comment (sulenchy) [reply index.html]: Sorry,  can't see any change. I suggested you add a valid value to the alt attribute. Like
```
<img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="image of friend4">
```

As you can see above, I set the value of the alt attribute to `image of friend4`.

File: login.html (hunk 4 above)
Comment (sulenchy) [reply login.html:42]: Looks good now

File: src/scss/styles.scss [hunk 8]
```diff
@@ -7,10 +7,10 @@
 .form-signin {
     max-width: 400px;
     padding: 15px;
-}
 
-.form-signin .form-floating:focus-within {
-    z-index: 2;
+    .form-floating:focus-within {
+            z-index: 2;
```
Comment (sulenchy) [inline src/scss/styles.scss:12]: This should be

```
&focus-within {
     z-insex: 2;
}
```

File: src/scss/styles.scss [hunk 9]
```diff
@@ -7,10 +7,10 @@
...
+    }
 }
 
 .form-signin input[type="email"] {
```
Comment (sulenchy) [inline src/scss/styles.scss:16]: This should be 

```
.form-signin {
 input[type="email"] {
    border-top-right-radius: 0;
 }
}
```

**Review by sulenchy — APPROVED** [review]
This looks good to me. Though, I requested some nice to have changed. You can implement them if you have some time else take note going forward. Cheers

**Pull request merged**